crates/typed-store/src/lib.rs

```diff
@@ -225,4 +230,14 @@ where
             .await
             .expect("Failed to receive reply to NotifyRead command from store")
     }
+
+    pub async fn iter(&self) -> HashMap<Key, Value> {
```

Comment:
shouldn't this return `impl Iterator<Item = (Key, Value)>` - as is this implies loading the whole table into memory.

Reply:
this was addressed offline